.github/workflows/update-pot-files.yml

```diff
@@ -0,0 +1,38 @@
+
+name: Update source *.pot files from source repo
+on:
+  schedule:
+    - cron: '45 0 * * *' # at 0:45 UTC
+
+  # allow to run the workflow manually
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Update *.pot files
+    runs-on: ubuntu-latest
+    # use pre-built Fedora container
+    container: ghcr.io/rpm-software-management/dnf-ci-host
+
+    steps:
+      - name: Checkout ci-dnf-stack
+        uses: actions/checkout@v2
+        with:
+          repository: rpm-software-management/ci-dnf-stack
+          path: ci-dnf-stack
```

Comment:
In the CI workflows the ci-dnf-stack is cloned directly into the current directory. Not sure how good of an idea that is ultimately... what about doing it the same way here for consistency? Or do you prefer a separate directory?

Reply:
Well, I do prefer separate directory - the reason was to avoid potential directory names collisions (e.g. when the `source` directory appeared in ci-dnf-stack repository).

Reply:
Yeah, that could happen... okay let's keep it this way.